tests/commands/test_store_commands.py

```diff
@@ -1063,6 +1065,73 @@ def test_status_with_multiple_branches(caplog, store_mock, config):
     assert expected == [rec.message for rec in caplog.records]
 
 
+def test_status_with_resources(caplog, store_mock, config):
+    """Support having multiple branches."""
+    caplog.set_level(logging.INFO, logger="charmcraft.commands")
+
+    res1 = Resource(name='resource1', optional=True, revision=1, resource_type='file')
+    res2 = Resource(name='resource2', optional=True, revision=54, resource_type='file')
+    channel_map = [
+        Release(revision=5, channel='latest/candidate', expires_at=None, resources=[res1, res2]),
+        Release(revision=5, channel='latest/beta', expires_at=None, resources=[res1]),
+    ]
+    channels = _build_channels()
+    revisions = [
+        _build_revision(revno=5, version='5.1'),
+    ]
+    store_mock.list_releases.return_value = (channel_map, channels, revisions)
+
+    args = Namespace(name='testcharm')
+    StatusCommand('group', config).run(args)
+
+    expected = [
+        "Track    Channel    Version    Revision    Resources",
```

Comment:
not related speficically two this, but the separation between headers/cloumns is supposed to be 2 spaces (https://discourse.ubuntu.com/t/tabular-data/18886)

Reply:
Opened #264 to fix all tables at once... we may need to stop using `tabulate` for this.